xcframework/src/main.rs

```diff
@@ -208,6 +197,18 @@ impl LibraryGroup {
 
         Ok(dest)
     }
+
+    fn headers_dir(&self) -> Result<PathBuf> {
+        let slice = self
+            .slices
+            .first()
```

Comment:
Use the first one here, because bindings in each slice (a.k.a target triple in cargo, `x86_64-apple-darwin`, `aarch64-apple-darwin`, etc.) should have the exact same content.

We could add a check here to ensure they are the same. Maybe it's an overkill? But I'm open to that.